Lib9c/Module/AdventureBossModule.cs

```diff
@@ -0,0 +1,121 @@
+using Libplanet.Action.State;
+using Libplanet.Crypto;
+using Nekoyume.Action;
+using Nekoyume.Action.Exceptions.AdventureBoss;
+using Nekoyume.Model.AdventureBoss;
+
+namespace Nekoyume.Module
+{
+    public static class AdventureBossModule
+    {
+        public static readonly Address LatestSeasonAddress = new($"{0:X40}");
+
+        private static string GetSeasonAsAddressForm(long season)
```

Comment:
```suggestion
        public static string GetSeasonAsAddressForm(long season)
```
AdventureBossModule 구현부분을 클라이언트에서도 동일하게 새로 작성해주어야해서 이부분 보호수준이 public이되는것이 좋을것같습니다~! @U-lis